### PR TITLE
Adding script to generate node configuration file.

### PR DESCRIPTION
**Pull request by ethack**

### CODE REVIEW

File: gen-node-cfg.sh [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+	fi
+} #End of askYN
+
+
+#======== Main ========
+
+PATH="/bin:/sbin:/usr/bin:/usr/sbin:$PATH"
+export PATH
+
+require_file /bin/awk /bin/cp /bin/date /bin/egrep /bin/sed /bin/tr /proc/cpuinfo /sbin/ip || exit 1
```
Comment (ethack) [inline gen-node-cfg.sh]: These tools can be at other locations. For instance, on my Ubuntu Server 16.04.3 system `awk` and `tr` are both in `/usr/bin`. Can we support if the tools are located anywhere in the user's path?

File: gen-node-cfg.sh [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+
+if [ -e "$bro_node_cfg" ]; then
+	cp -p "$bro_node_cfg" "$bro_node_cfg.$Now"
+	echo "A backup has been made for the existing $bro_node_cfg ."
+else
+	echo "$bro_node_cfg does not exist, creating one from scratch."
+fi
+
+
+cat "$this_script_path/node.cfg-template" | sed -e 's/@@InterfaceConfig@@/'"$node_configuration_block"'/' >"${bro_node_cfg}.tmp"
```
Comment (ethack) [inline gen-node-cfg.sh:182]: Could we move a check for the existence of node.cfg-template towards the beginning? If it doesn't exist (or can't be found), there isn't much point in continuing the script.

File: gen-node-cfg.sh (hunk 1 above)
Comment (william-stearns) [reply gen-node-cfg.sh]: Agreed - new version will look for these tools anywhere in the path.

File: gen-node-cfg.sh (hunk 2 above)
Comment (william-stearns) [reply gen-node-cfg.sh:182]: Agreed, new version will check that the file exists early.

**Pull request merged**